.github/ISSUE_TEMPLATE/mapping-request---issue.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Mapping request / issue
+about: Report a new request for a mapping or an issue with an existing mapping
+title: ''
+labels: ''
```

Comment:
Would having a default `mapping` label be helpful for searching/organizing issues?

Reply:
Good idea. Done.